.github/workflows/ci-systemd-release.yml

```diff
@@ -16,8 +16,15 @@ permissions: read-all
 
 jobs:
   goreleaser:
-    runs-on: ubuntu-22.04
-    if: github.repository == 'kubearmor/kubearmor'
+    runs-on: ${{ matrix.os }}
```

Comment:
Let's check if we can use enterprise runners here.

Reply:
any particular reason?

Reply:
not a bigger concern here right! it hardly takes ~4 minutes.